File: Getting-Practice-with-Python-and-Javascript/2- Control Flow Structures and Data Structures/Control Flow Structures/Javascript/Excercises/program-6.js
// 6-Escribir un programa que pida al usuario un número entero y muestre por pantalla si es múltiplo de 44.


let userInput = parseInt(prompt('Ingresa un numero entero: '))

if (userInput % 44 == 0) {
    console.log(`${true}, ${userInput} es multiplo de 44`)
} else {
    console.error(`${false}, ${userInput} no es multiplo de 44`)
}

File: Getting-Practice-with-Python-and-Javascript/2- Control Flow Structures and Data Structures/Control Flow Structures/Python/Excercises/program-6.py
# 6-Escribir un programa que pida al usuario un número entero y muestre por pantalla si es múltiplo de 44.


userInput = int(input('Ingresa un numero entero: '))

if userInput % 44 == 0:
    print(f'{True}, El numero {userInput} es multiplo de 44')
else:
    print(f'{False}, El numero {userInput} no es multiplo de 44')
